lte/gateway/c/core/oai/tasks/sgw/pgw_handlers.c

```diff
@@ -628,39 +620,38 @@ static int create_temporary_dedicated_bearer_context(
       sizeof(bearer_qos_t));
   // Save Policy Rule Name
   strcpy(eps_bearer_ctxt_p->policy_rule_name, bearer_req_p->policy_rule_name);
-
+  eps_bearer_ctxt_p->sgw_sequence_number = sequence_number;
   OAILOG_INFO_UE(
-      LOG_SPGW_APP, spgw_ctxt_p->sgw_eps_bearer_context_information.imsi64,
-      "Number of DL packet filter rules: %d\n",
+      module, sgw_ctxt_p->imsi64, "Number of DL packet filter rules: %d\n",
       eps_bearer_ctxt_p->tft.numberofpacketfilters);
 
   // Create temporary spgw bearer context entry
   pgw_ni_cbr_proc_t* pgw_ni_cbr_proc =
-      pgw_get_procedure_create_bearer(spgw_ctxt_p);
+      pgw_get_procedure_create_bearer(sgw_ctxt_p);
```

Comment:
although nit really part of this PR, could you take care of github action comment for line 630 and replace `strcpy` with `snprintf`